Packs/CortexXDR/ReleaseNotes/6_1_91.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Palo Alto Networks Cortex XDR - Investigation and Response
+
+- Added the argument **download_package** to command **xdr-get-distribution-url** to download an installation package file.
```

Comment:
please fix to:
Added support for the **download_package** argument in the **xdr-get-distribution-url** command to enable downloading an installation package file.